src/Annotator.js

```diff
@@ -337,10 +349,9 @@ class Annotator extends EventEmitter {
             return Promise.resolve({});
         }
 
-        return this.annotationService
-            .getThreadMap(this.fileVersionId)
+        return this.API.fetchVersionAnnotations(this.fileVersionId)
             .then((threads) => {
-                this.threadMap = threads;
+                this.apiAnnotations = threads;
```

Comment:
The name `apiAnnotations` is a little confusing. Should the local variable name be updated, as well?